seL4-platforms/platforms.yml

```diff
@@ -300,17 +300,9 @@ platforms:
     march: nehalem
 
 mcs_unsupported_platforms:
-# tx1 - scheduler accuracy fails, problem with clk rate in kernel or timestamp
-# at user level?
-# - TX1
-# zynq - scheduler accuracy fails, problem with clk rate in kernel or timestamp
-# at user level?
-- ZYNQ7000
-# not ported yet
-- ODROID_X
-- ZYNQMP
-- IMX8MM_EVK
-- IMX8MQ_EVK
-- MAAXBOARD # To be tested, for now this just copies the setting of IMX8MQ_EVK.
-- ROCKPRO64
-- ODROID_XU4
+  # fails SCHED0012
+  - IMX8MM_EVK
```

Comment:
Seems I lack some understanding how this is supposed to work. But why is this kept here now and the other have moved. Maybe you could add a comment about the background here, so this gets a bit more clear.

Reply:
Will add a comment. The idea is that this list here are the platforms where MCS doesn't work at all. The ones I've moved into build.py are platforms where MCS works for most configuration combinations, but not for some (those are then excluded).